### ISSUE TITLE
Emails sent to paper leads, project leads, working group

### ISSUE THREAD

**Issue by chee999** [LSSTDESC/desc-publications#67]
keep all the emails sent to the various groups (e.g. lead authors, project leads, working groups)

**Comment by sethdigel** [comment]
I agree.  Some of these e-mails will contain contribution statements (#70), and I am seeing that not only will the messages need to be kept, the contribution statements will need to be accessible.  They will need to be made available (eventually) in an online system promised in the Pub Policy.  And they may need to be edited sometimes by the Lead Authors for uniformity.  So I'm thinking that just a browsable e-mail archive probably will not be enough.

**Comment by chee999** [comment]
make the 'group','convener', etc. a link that will email all the members of that group. Individual links
should also be available for conveners, if one doesn't want to email the whole group.
